docs/using_eessi.md

```diff
@@ -0,0 +1,24 @@
+# Using EESSI
+
+There are different means to access EESSI depending on if and how it is set up on
+your system.
+
+### Determine whether EESSI is already set up on your system
+If you run the command _(Note, this may take a while to finish.)_
+```bash
+ls /cvmfs/pilot.eessi-hpc.org
+```
+and see a message like
+```
+2021.06  host_injections  latest  versions
```

Comment:
Let's remove the `2021.06` here, since that will get removed soon?

Reply:
Removed by https://github.com/EESSI/docs/pull/100/commits/0eae16fe58096e7b995d7e7d2ddc3d9208fc2001